### DISCUSSION TITLE
how can we add more than one inverter in a system for energy performance modelling

### DISCUSSION THREAD

**Discussion by hari0206** [pvlib/pvlib-python#1660] [Q&A] [answered]
how can we add more than one inverter in a system for energy performance modelling using pvlib

**Comment by cwhanse** [comment, marked as answer]
Currently, we can't. Each PVSystem has a single inverter, and each ModelChain has a single PVSystem. The workaround is to create one PVSystem for each inverter, and one ModelChain for each PVSystem, then run all ModelChains and add the AC power.

**Reply by hari0206** [reply]
@cwhanse Thank you for responding so quickly,
so does adding the ac power gives the accurate value ?
my use case is to perform modelling of a huge system like 45MW or 55MW so how can we achieve this

**Reply by hari0206** [reply]
@kanderso-nrel

**Reply by cwhanse** [reply]
Because pvlib's inverter models only predict true (active, real) AC power you can add AC power output from each inverter. pvlib has no capability for modeling reactive power.

As to accuracy for a very large system, that is likely more affected by the accuracy of the irradiance data, than any other factor. The desired input is the spatially-averaged irradiance on each array in the system. It is impractical to measure that quantity, so we substitute pyranometer measurements (1cm2) or values derived from satellite imagery (a few km2, and at minutes to hourly intervals). It is my impression that most large systems are built with power blocks of similar capacity, so it makes sense to model one power block (as a pvlib PVSystem), multiply the output by the number of similar power blocks, and then add in other PVSystems that represent atypical power blocks.
